the-unravelling/Assets/Scripts/World/BitmaskableWorldEntity.cs

```diff
@@ -0,0 +1,146 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+using UnityEngine.Tilemaps;
+
+
+[CreateAssetMenu(fileName = "Bitmaskable World Entity" ,menuName = "Scriptable Objects/World/Bitmaskable World Entity")]
+public class BitmaskableWorldEntity : WorldEntity
+{
+    private bool checkNorth;
```

Comment:
Might be cleaner to just keep these as a bitmask all the time, and not convert back and forth, but not a big deal as long as it works